QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+kube-system                       kindnet-l26qt                                     1/1     Running   0          85s
+kube-system                       kube-apiserver-florin-control-plane               1/1     Running   0          100s
+kube-system                       kube-controller-manager-florin-control-plane      1/1     Running   0          100s
+kube-system                       kube-proxy-qzhx6                                  1/1     Running   0          85s
+kube-system                       kube-scheduler-florin-control-plane               1/1     Running   0          99s
+local-path-storage                local-path-provisioner-684f458cdd-75wv8           1/1     Running   0          85s
+``` 
+
+### c. Create the nginx workload and deploy it to the florin cluster
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location object (`florin`) created earlier, thus directing the workload to `florin` edge cluster.
+
+In the `wmw-1` workspace create the following `EdgePlacement` object: 
+  
+```console
+kubectl ws root:my-org:wmw-1
+```
+
+```console
+  kubectl apply -f - <<EOF
+  apiVersion: edge.kcp.io/v1alpha1
+  kind: EdgePlacement
+  metadata:
+    name: edge-placement-c
+  spec:
+    locationSelectors:
+    - matchLabels: {"env":"prod"}
+    namespaceSelector:
+      matchLabels: {"common":"si"}
+    nonNamespacedObjects:
+    - apiGroup: apis.kcp.io
+      resources: [ "apibindings" ]
+      resourceNames: [ "bind-kube" ]
+    upsync:
+    - apiGroup: "group1.test"
+      resources: ["sprockets", "flanges"]
+      namespaces: ["orbital"]
+      names: ["george", "cosmo"]
+    - apiGroup: "group2.test"
+      resources: ["cogs"]
+      names: ["william"]
+  EOF
+```
+
+Deploy the nginx workload. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
+
+```console
+  kubectl apply -f - <<EOF
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: commonstuff
+    labels: {common: "si"}
+  ---
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: nginx-deployment
+    namespace: commonstuff
+    labels:
+      app: nginx
+  spec:
+    replicas: 3
+    selector:
+      matchLabels:
+        app: nginx
+    template:
+      metadata:
+        labels:
+          app: nginx
+      spec:
+        containers:
+        - name: nginx
+          image: nginx:1.14.2
+          ports:
+          - containerPort: 80
+  EOF
+  ```
+
+Check that the deployment was created in the florin edge cluster:
+
+```console
+KUBECONFIG=$florin_kubeconfig kubectl -n commonstuff get deployment
+```
+
+which should yield something like:
+
+```console
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+nginx-deployment   3/3     3            3           8m37s
+```
+
+Also, check that the pods are running in the florin edge cluster:
```

Comment:
The checking should first and foremost be that the deployed application is working.  Show a `curl` or `wget` that demonstrates this.